### DISCUSSION TITLE
[Bug]:

### DISCUSSION THREAD

**Discussion by Hardcore-fs** [pimcore/pimcore#15615] [General]
The  web pages error with a 500 error about writing directories in cache

### Pimcore version

11.0.5

### Steps to reproduce

following docker install here:
https://pimcore.com/docs/platform/Pimcore/Getting_Started/Installation/Docker_Based_Installation

picking this option:
# demo package with exemplary blueprints (`pimcore/demo`)
docker run -u `id -u`:`id -g` --rm -v `pwd`:/var/www/html pimcore/pimcore:php8.2-latest composer create-project pimcore/demo my-project

rocky linux 8
sestatus 
SELinux status:                 disabled

install as root...  (its only a fast demo)

### Actual Behavior

after fully installing there is an permission Denied error about not being able to write the internal docker  directories

docker exec -it  my-project-php-1 bash
entering into "/var/www/html"  ,  none of the directories are writable
if you then issue

chmod -R 777 /var/www

then suddenly the whole system works.

### Expected Behavior

it should work

**Comment by fashxp** [comment]
obviously there is something wrong with you file permissions...
see also our instructions: https://github.com/pimcore/demo#:~:text=Part%20of%20the,compose%20up%20%2Dd

**Comment by Hardcore-fs** [comment]
Clearly.....
but the instructions were followed to the letter, and it is the  docker that assembles the folder "my-project"
it is not like i setup the folder & docker puts the content in.
